### ISSUE TITLE
Use system flatbuffer in standalone-build OSS CI

### ISSUE THREAD

**Issue by zasdfgbnm** [NVIDIA/Fuser#950]
The runner is super slow, building flatbuffer takes 10min.

**Comment by nWEIdia** [comment]
Is system flatbuffer compatible with 23.3.3 requirement? In one case, the system installed "libflatbuffers-devel" was only 1.11.0 (https://packages.ubuntu.com/focal/libflatbuffers-dev)

**Comment by zasdfgbnm** [comment]
The generated code is removed in https://github.com/NVIDIA/Fuser/pull/965, so we are not requiring a specific version now. However, 1.11.0 sounds like such an old version, I am not sure if it will be compatible.

**Comment by kevinstephano** [comment]
@zasdfgbnm or @rdspring1 is this still an issue?

**Comment by zasdfgbnm** [comment]
Yes, this is related to GitHub runner time. Today, we no longer require GitHub runner to pass before merging, so it is urgent. But it is always better to make the runner faster, so I would not consider this as irrelevant.
